ext/pg_result.c

```diff
@@ -390,6 +395,27 @@ pgresult_get(VALUE self)
 	return this->pgresult;
 }
 
+static VALUE pg_cstr_to_sym(char *cstr, unsigned int flags, int enc_idx)
```

Comment:
The signature of this function could be more clear that it is specific to field names.

Reply:
Shrug! I don't care much about static function names.